tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c

```diff
@@ -90,7 +90,7 @@
 #elif CONFIG_SOC_RISCV_TELINK_B95
 #include <ext_driver/driver_lib/ext_lib.h>
 #elif CONFIG_SOC_RISCV_TELINK_TL321X
-#include <ext_driver/driver_lib/ext_lib.h>
+#include <ext_driver/driver_internal/ext_lib.h>
 #endif
 #include <ext_driver/ext_misc.h>
```

Comment:
Looks like you just copied code from here 
In case if you are using separate  ecp_alt file for TLX SoC, remove it from here and in TLX don't use others SoCs

Reply:
@pulin1103 @haiwentelink 
I'm not sure if we need to create a new file such as ecp_alt_tlx.c for TLX.
Could you please check what we can do with it?

Reply:
This branch will not be merged into the develop branch for now. The MbedTLS HW acceleration will continue using the version prior to Andrii's optimization to facilitate ongoing development. The TL321x MbedTLS HW acceleration adaptation will be made before merging into the develop branch later.

Reply:
Okay. 
I see @andriy-bilynskyy 's commit (e5a21d1fb577c339ca149db91a67764e00338c61) on develop branch.
It is unnecessary to create separate ecp_alt temporarily if we need to adjust this module later as Andriy did.